src/common/SPELLS/classic/warrior.ts

```diff
@@ -1,7 +1,262 @@
 import { spellIndexableList } from '../Spell';
 
 const spells = spellIndexableList({
-  VIGILANCE: {
+  // --------
+  // SHARED
+  // --------
+ 
+Battle_Shout: {
```

Comment:
All object keys need to be in all CAPS to follow style format.
So this one would be changed to `BATTLE_SHOUT`.
name values use title case